docs/content/latest/yugabyte-cloud/cloud-faq.md

```diff
@@ -143,9 +137,12 @@ For multi-region deployments, including [synchronous replication](../../explore/
 
 Upgrades are automatically handled by Yugabyte. There are two types of upgrades:
 
-- Cloud console - During a maintenance window, Yugabyte Cloud console may be in read-only mode and not allow any edit changes. The upgrade has no impact on running clusters. Customers will be notified in advance of the maintenance schedule.
+Cloud console
+: During a maintenance window, the Yugabyte Cloud console may be in read-only mode and not allow any edit changes. The upgrade has no impact on running clusters. Customers will be notified in advance of the maintenance schedule.
 
-- Cluster (yugabyteDB) version upgrade - To keep up with the latest bug fixes, improvements, and security fixes, Yugabyte will upgrade your cluster to the latest version. We will notify you of any upcoming upgrade schedule via email and Slack. All database upgrades are done on a rolling basis to avoid any downtime.
+Cluster (yugabyteDB) version upgrade
+: To keep up with the latest bug fixes, improvements, and security fixes, Yugabyte will upgrade your cluster to the latest version. We will notify you of any upcoming upgrade schedule via email and Slack.
```

Comment:
Slack??? How does that work? That seems like an internal thing, not a paid-customers thing. But if so, mention what Slack workspace, and what channel...